web/index.html

```diff
@@ -203,7 +203,6 @@ <h1 id="title">The 2022 /r/place Atlas</h1>
 					<p>Please copy the text below and submit it as a<br>
 					new text post to <a target="_blank" href="https://www.reddit.com/r/placeAtlas2/">/r/placeAtlas2</a> on Reddit.</p>
 					<p>I will then check it and add it to the atlas.</p>
```

Comment:
Please remove the duplicate text as well 
Also if you can, make <a> go directly to reddit and move it to the same div as button below

Reply:
In my last PR, I moved it on the top so people won't miss it.

Yes, the dupe text shouldn't been needed (in fact just revert to the text I made excluding the warning)

Reply:
Is this ok?
![imagen](https://user-images.githubusercontent.com/24706838/162348050-967d8cc2-861d-43ce-9dad-f1e474ad7d41.png)

Reply:
Last time I also add a text just so people won't get confused, but that is my minimum good.

Reply:
Thanks for the warning, added back the same message